### PR TITLE
Initial Test Set Up

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Add RSpec
* Add 'Log in' Spec
* Add 'Introduction' Spec
* Add travis config
* Add rake

[#118409261]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.bundle
```
Comment (ericcf) [inline .gitignore]: Don't forget `.DS_Store`.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+.rspec
+.DS_Store.
```
Comment (ericcf) [inline .gitignore]: I don't think you want the `.` at the end. There are still `.DS_Store` files in this commit.

**Comment by ericcf** [comment]
Nice work so far! Just some minor cleanup with `.gitignore` and it looks good.

**Comment by michaelwehrley** [comment]
Thanks @ericcf

**Pull request merged**